lib/Service/SampleConversationsService.php

```diff
@@ -78,7 +78,7 @@ public function initialCreateSamples(string $userId): void {
 	protected function defaultSampleConversation(IUser $user): void {
 		$room = $this->roomService->createConversation(
 			Room::TYPE_GROUP,
-			$this->l->t('Let´s get started!'),
+			$this->l->t('Let's get started!'),
```

Comment:
This breaks code parsing, as you're closing the line. Try to escape a character instead:
```suggestion
			$this->l->t('Let\'s get started!'),
```

Reply:
@Antreesy Thanks. I forgot about that.